tests/fixtures/issues/issue-000-invalids.php

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
This is my idea for the Invalid logging that I mention in #1291. I think this format keeps it more succinct and readable (compared to having the default properties block inserted into each agent like the other tests cases are).

A few of these come from a couple of existing test cases (I removed them from there) as they were agents that I ran into when looking through the attached files. 

I left the `issue-000-invalids-A` key names because I felt that would make it easier to reference which agent failed a test from PHPUnit's output.